wordle-web/src/views/HomeView.vue

```diff
@@ -1,9 +1,52 @@
-<template>
-  <main>
-    <TheWelcome />
-  </main>
-</template>
-
 <script setup lang="ts">
-import TheWelcome from '../components/TheWelcome.vue'
+import name_and_logo from '@/assets/Xela_Games.png'
+import 'animate.css'
 </script>
+
+<script lang="ts">
+export default {
+  data: () => ({
+    showImage: false
+  })
+}
+</script>
+
+<template id="body">
+  <h1 id="h1">
+    Welcome to . . . <v-btn color="secondary" @click="showImage = !showImage">CLICK ME</v-btn>
```

Comment:
there isn't an obvious way to make my way from the welcome page to the wordle page from here. I suggest you make a btn appear which takes you to the game post logo reveal.